.gitignore

```diff
@@ -1,3 +1,4 @@
 .rvmrc
 .bundle
 .tool-versions
+.DS_Store
```

Comment:
This is fine, and also the repetitiveness of doing it for every single repo can be avoided by adding it to your home gitignore:
```bash
echo ".DS_Store" > $HOME/.gitignore
```

Reply:
Thanks!